src/guest_webapp/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# choose node
+FROM node:20-alpine AS build_react
+
+# set working directory inside the container
```

Comment:
(this is personal preference, but)
Most of these comments might not be necessary since they describe how dockerfile commands work, rather than adding additional information. Some reasons to add a comment could include:
- visually breaking distinct sections of the dockerfile (eg, new stages)
- adding additional, unavailable information (eg, environment variable alternatives)